packages/lesswrong/components/notifications/NotificationsList.jsx

```diff
@@ -15,17 +24,16 @@ class NotificationsList extends Component {
   }
 
   render() {
-    const results = this.props.results;
-    const loadMore = this.props.loadMore;
+    const { results, loadMore, classes } = this.props;
     if (results && results.length) {
       return (
         <List style={{width: '270px', overflowY: 'auto', padding: '0px'}}>
           {results.map(notification => <Components.NotificationsItem notification={notification} lastNotificationsCheck={this.state.lastNotificationsCheck} key={notification._id} />)}
-          {results.length >= 20 && <ListItem className="notifications-list-item" onClick={() => loadMore()} primaryText="Load More" style={{textAlign: 'center', fontSize: '14px'}} />}
+          {results.length >= 20 && <ListItem className={classes.listItem} onClick={() => loadMore()} primaryText="Load More" style={{textAlign: 'center', fontSize: '14px'}} />}
         </List>
       )
     } else {
-      return <div className="notifications-list-empty"> You don{"'"}t have any notifications yet!</div>
+      return <div className={classes.empty}> You don{"'"}t have any notifications yet!</div>
```

Comment:
I know this wasn't the main part of the PR but can you fix this weird apostrophe situation while it's drawn to our attention?